cmp/74525bd1-3387-4bd6-ab0d-ed3969b6048b/component.lp

```diff
@@ -0,0 +1,53 @@
+(librepcb_component 74525bd1-3387-4bd6-ab0d-ed3969b6048b
+ (name "ATtiny13A")
+ (description "Atmel 8-bit AVR Microcontroller with 1 kilobyte in-system programmable flash.")
+ (keywords "attiny,atmel,avr,8bit")
+ (author "steve")
+ (version "0.1")
+ (created 2020-05-15T00:41:23Z)
+ (deprecated false)
+ (category 22151601-c2d9-419a-87bc-266f9c7c3459)
+ (schematic_only false)
+ (default_value "{{PARTNUMBER or DEVICE or COMPONENT}}")
+ (prefix "U")
+ (signal de47a3ac-60bc-4571-8334-b7efd503b2cf (name "PB4/PCINT4/ADC2") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 1aa108ab-bbfc-4fa5-9862-324acad0b10f (name "GND") (role passive)
+  (required true) (negated false) (clock false) (forced_net "")
+ )
+ (signal a0dc9e3a-c2a9-4389-b1e9-261be6d9f70f (name "PB0/MOSI/AREF/OC0A/PCINT0") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
+ (signal 9264c42b-76c4-4b0a-8ac4-1204371363e8 (name "PB1/MISO/OC0B/INT0/PCINT1") (role passive)
+  (required false) (negated false) (clock false) (forced_net "")
+ )
```

Comment:
Until the issue with the overlapping pin labels is resolved (see #11), I'd skip the counter/timer functionality:

```suggestion
 (signal a0dc9e3a-c2a9-4389-b1e9-261be6d9f70f (name "PB0/MOSI/AREF/PCINT0") (role passive)
  (required false) (negated false) (clock false) (forced_net "")
 )
 (signal 9264c42b-76c4-4b0a-8ac4-1204371363e8 (name "PB1/MISO/INT0/PCINT1") (role passive)
  (required false) (negated false) (clock false) (forced_net "")
 )
```

That should avoid the issue for now.